tests/sentry/seer/test_utils.py

```diff
@@ -156,6 +156,7 @@ def test_empty_similar_issues_embeddings(mock_seer_request, default_project):
 
     params: SimilarIssuesEmbeddingsRequest = {
         "group_id": NonNone(event.group_id),
+        "group_hash": NonNone(event.get_primary_hash()),
```

Comment:
just to confirm - we're changing this (and all other `group_hash` occurrences) to just `hash` in a follow-up PR and accepting that the similarity endpoint will be a bit broken until that happens?

Reply:
On the name change, yes, that's the next PR to come. As for things being broken... I thought we said nothing was currently relying on `group_hash`? In any case, I'm happy to hold off on merging this until the rename is ready.

Reply:
yeah nothing's relying on `group_hash` but we're also not expecting it as part of the request right now on the seer side (my PR only will accept `hash).` I can make that change if needed.

Reply:
Will it error out or just ignore it? (I was assuming the latter.) And Seer's not yet requiring the hash, correct?

Reply:
oh good point, I think you're right. are you able to test it with the current seer implementation (since we're also not accepting `group_hash`)? 

and correct, seer is not requiring the hash yet.

Reply:
[update]: seer just ignores the unneeded parameter